src/ray/common/task/task_spec.cc

```diff
@@ -47,11 +47,9 @@ SchedulingClass TaskSpecification::GetSchedulingClass(
     sched_cls_id = ++next_sched_id_;
     // TODO(ekl) we might want to try cleaning up task types in these cases
     if (sched_cls_id > 100) {
```

Comment:
question: why we chose to check 100 or 1000? can we make the parameters configurable (e.g., listing the parameters in a configuration file so that we could easily check those parameters)?

Reply:
I don't know why we pick this particular number. This is code written multiple years ago. I don't think we need to make it configurable, it just prints out a warning so changing it doesn't have a real impact.

Reply:
Ideally we shouldn't have hard-coded value in the code. This is a separate issue though so let us fix one thing at a time :-)